client/src/services/batchList/reducer.tsx

```diff
@@ -0,0 +1,35 @@
+import { ISimpleAction, Batch, BatchResponse } from 'types';
+
+const initialState: BatchResponse = {
+  data: {},
+  error: null
+};
+
+const batchList = (state = initialState, action: ISimpleAction): any => {
+  switch (action.type) {
+    case 'LIST_BATCH_SUCCESS': {
+      return {
+        data: action.payload
```

Comment:
Why is this ternary needed?

Reply:
The api returns null when there's no result (instead of an empty array). Maybe we should change that in another pr.